vfs/vfscache/cache.go

```diff
@@ -195,18 +195,19 @@ func createRootDirs(parentOSPath string, relativeDirOSPath string) (dataOSPath s
 //
 // Returns an os path for the data cache file.
 func (c *Cache) createItemDir(name string) (string, error) {
-	parent := vfscommon.FindParent(name)
-	parentPath := c.toOSPath(parent)
-	err := createDir(parentPath)
-	if err != nil {
+	relativePath := toOSPath(name)
```

Comment:
I'm not sure this change is worth doing - I don't think it add significant clarity to the code, nor does it make it any quicker?